docs/data-sources/server.md

```diff
@@ -124,4 +124,7 @@ The following attributes are returned by the datasource:
     * `port_range_start` - Defines the start range of the allowed port (from 1 to 65534) if protocol TCP or UDP is chosen
     * `port_range_end` - Defines the end range of the allowed port (from 1 to 65534) if the protocol TCP or UDP is chosen
     * `type` - The type of firewall rule
-  
\ No newline at end of file
+* `labels` - list of
+  * `id` - The Id of the label
```

Comment:
**NOTE**: This doesn't add any extra or useful information about the attributes, but I wrote them in order to keep the consistency of the file since I noticed that for other attributes we did the same.